graphdatascience/graph_data_science.py

```diff
@@ -18,20 +18,21 @@ def __init__(
         self, endpoint: Union[str, QueryRunner], auth: Any = None, aura_ds: bool = False
     ):
         if isinstance(endpoint, str):
-            self._config = {}
+            self._config: Dict[str, Any] = {"user_agent": "neo4j-gds-client"}
```

Comment:
I think we should use the official name in this place, which would be like this
```suggestion
            self._config: Dict[str, Any] = {"user_agent": "neo4j-graphdatascience"}
```

I also wonder whether it would be cool to also include a version part, but not sure how to feed it here from the `setup.py`. Then the `AuraDS` data would include information about how up to date our users are.

Reply:
Done